american-truck-simulator.kvp

```diff
@@ -0,0 +1,98 @@
+Meta.DisplayName=American Truck Simulator
+Meta.Description=American Truck Simulator Dedicated Server
+Meta.OS=Windows, Linux
+Meta.AarchSupport=NotSupported
+Meta.Arch=x86_64
+Meta.Author=IceOfWraith
+Meta.URL=https://store.steampowered.com/app/270880/American_Truck_Simulator/
+Meta.DisplayImageSource=steam:270880
+Meta.EndpointURIFormat=steam://connect/{ip}:{GenericModule.App.Ports.$SteamQueryPort}
+Meta.ConfigManifest=american-truck-simulatorconfig.json
+Meta.MetaConfigManifest=american-truck-simulatormetaconfig.json
+Meta.ConfigRoot=american-truck-simulator.kvp
+Meta.MinAMPVersion=2.4.3.0
+Meta.SpecificDockerImage=
+Meta.DockerRequired=False
+Meta.ContainerPolicy=SupportedOnLinux, RequiredOnWindows
```

Comment:
RequiredOnWindows does not actually ... require docker on Windows

Reply:
I know, I just put things to make it look good. I'm weird like that.